frontend/src/metabase-types/api/mocks/entity-id.ts

```diff
@@ -0,0 +1,5 @@
+import type { BaseEntityId } from "../entity-id";
+
+export const createMockEntityId = (): BaseEntityId => {
+  return "abcdefghijklmnopqrstu" as BaseEntityId;
```

Comment:
nit: I'd expect this function to return something unique. This might not be apparent now, but when we need to have different entity_id for reference this could make tests behave unexpectedly. I think it's fine for now if we don't want to do anything about it.

But can you at least have a todo comment that this needs to either:
1. Be implemented or
2. Use the ID generation function from BE (which isn't exported currently)

Reply:
I'll just create a simple generator right now and add a TODO. Maybe we can move this to CLJS in the future

Reply:
ah! we can use the `nanoid` package supplied by rtk for now 😄 much better